tx-parser/benches/bench.rs

```diff
@@ -0,0 +1,55 @@
+#![feature(test)]
+
+// These benchmarks can be run from the project root with:
+// > cargo bench --package mentat_tx_parser
+
+extern crate test;
+extern crate edn;
+extern crate mentat_tx_parser;
+
+use test::Bencher;
+use mentat_tx_parser::Tx;
+
+#[bench]
+fn bench_parse1(b: &mut Bencher) {
+    let input = r#"[[:db/add 1 :test/val "a"]]"#;
+    let parsed_edn = edn::parse::value(input).expect("to parse test input");
+    b.iter(|| {
+        return Tx::parse(parsed_edn.clone());
```

Comment:
Note: on those numbers I had only made the change to bench_parse2 so ignore bench_parse1